application/ui/tsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
     "extends": "@geti/config/typescript",
-    "include": ["src", "tests", "rsbuild.config.ts", "vitest.config.ts", "playwright.config.ts"],
+    "include": ["src", "tests", "rsbuild.config.ts", "vitest.config.ts", "playwright.config.ts", "mocks"],
     "compilerOptions": {
         "types": ["vitest/globals"],
         "baseUrl": ".",
         "paths": {
-            "@test-utils/*": ["./src/test-utils/*"]
+            "@test-utils/*": ["./src/test-utils/*"],
+            "@hooks/*": ["./src/hooks/*"],
+            "@mocks/*": ["./mocks/*"],
```

Comment:
Personal note: I don't like using absolute imports for these things. 

I'd prefer to either use a single absolute import like `"~/*": ["./src"]` (that is then used by all apps) or to stick with relative imports.
My reasoning for this is that having a lot of different path aliases makes it hard to reason/read about the code because you'd need to know what is inside of the `tsconfig` file.